beets/ui/commands.py

```diff
@@ -824,11 +824,19 @@ def resolve_duplicate(self, task, found_duplicates):
         log.warn(u"This {0} is already in the library!",
                  (u"album" if task.is_album else u"item"))
 
+        # If set, get the action to follow from config.
+        sel = config['import']['duplicate_action'].as_choice({
+            u'skip': u's',
+            u'keep': u'k',
+            u'remove': u'r',
+            u'ask': None,
+        })
+
```

Comment:
Would you mind moving this check into `importer.py` instead of `commands.py`, in [the `resolve_duplicates` function](https://github.com/beetbox/beets/blob/a76de9518318e6d401585c8a69f84d5da7085a9a/beets/importer.py#L1320-L1331)?

The idea with this separation is that all the UI-agnostic code, stuff that can be decided based *only* on the configuration file, goes in the generic importer infrastructure. Only the components that interact with the user go in the CLI code.